library/std/src/primitive_docs.rs

```diff
@@ -100,7 +100,7 @@ mod prim_bool {}
 /// behaviour of the `!` type - expressions with type `!` will coerce into any other type.
 ///
 /// [`u32`]: prim@u32
-/// [`exit`]: process::exit
```

Comment:
I don't think this link should've worked before; `std::process` is never imported. It must be a bug in how primitives are/were implemented. I think switching to using the `../` style links for `std` too may fix the errors.

Reply:
This file isn't a module, it uses `include!` directly, which means it has the scope of the crate root. So since the crate root for libstd has `mod process`, process is in scope.

Reply:
Are primitives resolved in the scope of the crate root rather than the module they're defined in?

Reply:
I think you misunderstood - this is included from the crate root. The module they're defined in *is* the crate root. https://github.com/rust-lang/rust/blob/6d64f7f695943541fe12bb960971403f440d7225/library/std/src/lib.rs#L572

Reply:
Oh! I didn't realized that. I always assumed that primitives showed up at the crate root because of some special rustdoc behavior; I didn't realize that it was just an `include!`. Thanks :)